xbmc/video/tags/VideoInfoTagLoaderFactory.cpp

```diff
@@ -38,7 +40,8 @@ IVideoInfoTagLoader* CVideoInfoTagLoaderFactory::CreateLoader(const CFileItem& i
     return nfo;
   delete nfo;
 
-  if (item.IsType(".mkv") || item.IsType(".mp4"))
+  if (CServiceBroker::GetSettings().GetBool("myvideos.usetags") &&
```

Comment:
We usually add a string constant for settings here: https://github.com/xbmc/xbmc/blob/master/xbmc/settings/Settings.h + https://github.com/xbmc/xbmc/blob/master/xbmc/settings/Settings.cpp

Reply:
I guess i will have to comply with this horrid idea ;)

Reply:
added.